src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -64,7 +64,12 @@ export class AddBulkConcepts extends Component {
       <div className="container-fluid add-bulk-concepts custom-max-width">
         <Header locationPath={this.props.match.params} />
         <h3>
-          <strong>{dictionaryName} Dictionary</strong>: Bulk Add Concepts
+          <strong>
+            {dictionaryName}
+            {' '}
```

Comment:
What is the above line 69 for?

Reply:
It is used to put an explicit space in a text block, eslint --fix replaced the space that was there before with this. It is a way to put a space in the text block.

Reply:
Did you evaluate the HTML tag for a non-breaking space?

Reply:
yes I did, but this was used because it's in jsx.